### DISCUSSION TITLE
Change the port from 3000 or 3001 to 80 on LXCs like Homarr or Uptime Kuma

### DISCUSSION THREAD

**Discussion by DorjeDorf** [tteck/Proxmox#2972] [Help Needed]
I am loving the scripts and they have certainly made my life so much easier. Thank you for those. 

I have been wondering, however, about the seemingly default port for most WebUIs offered by the scripts here. They are usually 3000 or 3001 - even when the original apps or their docker images offer different ports. 

Is there an easy way to change the ports to default 80? 
Granted, I've only just started with Proxmox, so I might not know enough, but from what I see - LXCs get their own IPs, so we don't need to worry about port sharing and can instead use the default 80 for most things. 

How can that be achieved? 

With homarr, I tried editing the .env file in /opt/homarr, but that didn't help. 

Any hints would be most appreciated.

**Comment by crazyduck** [comment]
I would like to change the dashy port to 80, where is the default port 3000 set during install? As I can find docker tutorials only...

**Comment by Mellowlynx** [comment]
Different apps use different config options, sometimes it's easy to change it, and sometimes it can't be changed at all.
Some apps have the port almost hard code intro them, so changing it is not an option.

I like to suggest to start using a reverse proxy.
You can set it up that **_status.yourdomain.tld_** is your Uptime kuma and **_start.yourdomain.tld_** is your Homer page.

**Comment by faithless159** [comment]
### **For Homarr**
at the end of .env file write PORT="80".
also chage NEXTAUTH_URL="http://localhost:3000/" to NEXTAUTH_URL="http://localhost/"
worked for me

DATABASE_URL="file:./database/db.sqlite"
NEXTAUTH_URL="http://localhost:3000/"
NEXTAUTH_SECRET="dsamdsaliWMDLamwdwada48d34a34dsa3w8d4"
NEXT_PUBLIC_DISABLE_ANALYTICS="true"
DEFAULT_COLOR_SCHEME="dark"
PORT="80"
